server/sonar-web/src/main/js/apps/sessions/routes.js

```diff
@@ -0,0 +1,31 @@
+/*
+ * SonarQube
+ * Copyright (C) 2009-2016 SonarSource SA
+ * mailto:contact AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+import React from 'react';
+import { Route, Redirect } from 'react-router';
+import LoginFormContainer from './components/LoginFormContainer';
+import Logout from './components/Logout';
+import Unauthorized from './components/Unauthorized';
+
+export default [
+  <Redirect key="login" from="/sessions/login" to="/sessions/new"/>,
+  <Route key="new" path="new" component={LoginFormContainer}/>,
+  <Route key="logout" path="logout" component={Logout}/>,
+  <Route key="unauthorized" path="unauthorized" component={Unauthorized}/>,
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this trailing comma. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=javascript%3ATrailingComma)